### PR TITLE
Fix/issue#367

### PR DESCRIPTION
**Pull request by ganeshpatro321**
I mistakenly deleted the previous branch, so I have created another PR, please look into it.
@jingchunzhu

### CODE REVIEW

**Comment by jingchunzhu** [comment]
Code does not work. Please try your code on study "GDC Pan-Cancer (PANCAN)" to verify that it works before sending pull request.

File: js/views/Column.js [hunk 1]
```diff
@@ -27,6 +27,8 @@ import { matches } from 'static-interval-tree';
 var gaEvents = require('../gaEvents');
 var crosshair = require('./cursor.png');
 
+import { fetchCohortTumorMeta as tumorMapLinkoutMeta }  from '../xenaQuery';
+
 var ESCAPE = 27;
```
[the remark this answers is not in the document]
Comment (ganeshpatro321) [reply js/views/Column.js]: May be for better UX experience for client.

**Comment by ganeshpatro321** [comment]
I will check with that study.

File: js/xenaQuery.js [hunk 2]
```diff
@@ -465,5 +467,6 @@ module.exports = {
 	// cohort meta
 	fetchCohortMeta: fetchJSON(cohortMetaURL),
 	fetchCohortPreferred: fetchJSON(cohortPreferredURL),
```
Comment (ganeshpatro321) [inline js/xenaQuery.js]: Hey @jingchunzhu Can I know where are these used? fetchCohortMeta or fetchCohortPreffered.

**Comment by ganeshpatro321** [comment]
Hey @jingchunzhu 

I tried to use the fetchJSON function which was already defined in the xenaQuery file, but that was returning an observable object with body undefined.
I tried to search for data which are been fetched using the fetchJSON function, but couldn't find the usability of the data anywhere.

So, I used a fetch() call which will work similar, and fetch the object, and I am setting the variable to the object.

Please review the code and if possible send the link where I can check where this object is been rendered or used.

Thank You.

**Comment by jingchunzhu** [comment]
We use Rx for Ajax calls in Xena. You could use the javascript Fetch API, but for consistency, we prefer Rx.  Declare a global variable to store tumormap query results is not a good idea.

**Comment by ganeshpatro321** [comment]
Okay, I could not catch how the code was fetching the JSON using Rx. Anyways thank you, I will look into it.
@jingchunzhu

**Pull request closed without merging**